lib/account/report_issue.dart

```diff
@@ -226,9 +227,8 @@ class _ReportIssueState extends State<ReportIssue> {
         },
       );
     } catch (error, stackTrace) {
-      if (Platform.isIOS) {
-        AppLoadingDialog.dismissLoadingDialog(context);
-      }
+      AppLoadingDialog.dismissLoadingDialog(context);
```

Comment:
Whenever I report an issue now, I am seeing a loading dialog with a Lantern logo:

![Screenshot 2024-03-26 at 7 09 03 AM](https://github.com/getlantern/lantern-client/assets/6346213/03a48bf6-75cb-421e-8f64-745667f07366)

Is that expected?

Reply:
Yes, That is expected